devicetypes/smartthings/aeon-multisensor.src/aeon-multisensor.groovy

```diff
@@ -110,17 +122,32 @@ def parse(String description)
 	def result = []
 	def cmd = zwave.parse(description, [0x31: 2, 0x30: 1, 0x84: 1])
 	if (cmd) {
-		if( cmd.CMD == "8407" ) { result << new physicalgraph.device.HubAction(zwave.wakeUpV1.wakeUpNoMoreInformation().format()) }
 		result << createEvent(zwaveEvent(cmd))
 	}
 	log.debug "Parse returned ${result}"
 	return result
 }
 
 // Event Generation
+def zwaveEvent(physicalgraph.zwave.commands.securityv1.SecurityMessageEncapsulation cmd) {
+	def encapsulatedCommand = cmd.encapsulatedCommand([0x31: 2, 0x30: 1, 0x84: 1])
+	if (encapsulatedCommand) {
+		zwaveEvent(encapsulatedCommand)
+	} else {
+		log.warn "Unable to extract encapsulated cmd from $cmd"
+		createEvent(descriptionText: cmd.toString())
+	}
+}
+
 def zwaveEvent(physicalgraph.zwave.commands.wakeupv1.WakeUpNotification cmd)
 {
-	[descriptionText: "${device.displayName} woke up", isStateChange: false]
+	def cmds = []
+	def result = createEvent(descriptionText: "$device.displayName woke up", isStateChange: false)
+	if (!state.lastbatt || (now() - state.lastbatt) >= 10 * 60 * 60 * 1000) {
```

Comment:
Going forward and Per Z-Wave Plus V2 rules, we need to ask all the sleepy nodes for their battery level when they wake up. So I would say, let's just keep it simple, and just ask for Battery level every time the device wakes up